helios-ts/src/storage.rs

```diff
@@ -0,0 +1,45 @@
+extern crate console_error_panic_hook;
+extern crate web_sys;
+
+use config::Config;
+use consensus::database::Database;
+use eyre::Result;
+
+#[derive(Clone)]
+pub struct StorageDB {
+    checkpoint: Vec<u8>,
+}
+
+impl Database for StorageDB {
```

Comment:
Can we rename this to `LocalStorageDB` to make it clear that it is the browser local storage.